README.rst

```diff
@@ -94,11 +94,10 @@ Here are parameters of the **ScaleBar** class constructor.
 * ``dimension``: dimension of *dx* and *units*.
   It can either be equal
 
-  * ``SI_LENGTH``: scale bar showing km, m, cm, etc.
-  * ``IMPERIAL_LENGTH``: scale bar showing in, ft, yd, mi, etc.
-  * ``SI_LENGTH_RECIPROCAL``: scale bar showing 1/m, 1/cm, etc.
-  * ``PIXEL_LENGTH``: scale bar showing px, kpx, Mpx, etc.
-  * a ``matplotlib_scalebar.dimension._Dimension`` object
+  * ``si-length``: scale bar showing km, m, cm, etc.
```

Comment:
I agree this was confusing. SI_LENGTH actually refers to a module variable, which could have been imported along the ScaleBar class, but you are write I am sure most people uses the string.